core/src/main/resources/META-INF/resources/primefaces-extensions/clockpicker/1-clockpicker-widget.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * PrimeFaces Extensions ClockPicker Widget.
+ * 
+ */
+PrimeFaces.widget.ExtClockPicker = PrimeFaces.widget.BaseWidget.extend({
```

Comment:
You can just make this `ClockPicker` not `ExtClockPicker` if you want.

Reply:
Yes, I checked the rest of the components but they all have Ext as the prefix. Should we keep Ext for consistency purposes?

For example,

https://github.com/primefaces-extensions/primefaces-extensions/blob/927f6b55320df431d91bb9148deac2489adb6c2e/core/src/main/resources/META-INF/resources/primefaces-extensions/inputphone/inputphone-widget.js#L7